traffic_portal/app/src/common/api/ServerService.js

```diff
@@ -140,7 +140,15 @@ var ServerService = function($http, locationUtils, messageModel, ENV) {
     };
 
     this.getDeliveryServiceServers = function(dsId) {
-        return $http.get(ENV.api['root'] + 'deliveryservices/' + dsId + '/servers').then(
+        return $http.get(ENV.api['root'] + 'servers?dsId=' + dsId).then(
+            function (result) {
+                return result.data.response;
+            }
+        );
+    };
+
+    this.getTopologyServers = function(topology) {
+        return $http.get(ENV.api['root'] + 'servers?topology=' + topology).then(
```

Comment:
that'll need to be url-encoded - which is easily accomplished by just passing it in the `params` instead of constructing a URL with a query string.